src/contrib/ethosu/cascader/block_config.cc

```diff
@@ -37,15 +37,17 @@ void BlockConfigNode::VisitAttrs(AttrVisitor* v) {
   v->Visit("_input_shape", &tmp_arr);
   tmp_arr = make_array(output_shape_);
   v->Visit("_output_shape", &tmp_arr);
+  v->Visit("_compute_cycles", &compute_cycles_);
+  v->Visit("_output_cycles", &output_cycles_);
 }
 
 BlockConfig::BlockConfig(const std::vector<int>& input_shape, const std::vector<int>& output_shape,
                          int compute_cycles, int output_cycles) {
   auto n = make_object<BlockConfigNode>();
   n->input_shape_ = std::move(input_shape);
   n->output_shape_ = std::move(output_shape);
-  n->compute_cycles_ = compute_cycles;
-  n->output_cycles_ = output_cycles;
+  n->compute_cycles_ = std::move(compute_cycles);
+  n->output_cycles_ = std::move(output_cycles);
```

Comment:
There is no need for `std::move` for integers.

Reply:
You're right, I removed the `std::move`